homeassistant/components/fitbit/sensor.py

```diff
@@ -449,7 +449,7 @@ def update(self) -> None:
                     self._attr_native_value = raw_state
                 else:
                     try:
-                        self._attr_native_value = f"{int(raw_state):,}"
+                        self._attr_native_value = f"{int(raw_state)}"
```

Comment:
This is a breaking change for users parsing the state string, right? Should we ignore that and release this as a bug fix (in a patch release) or should we release it as a breaking change in the February release?

Reply:
You're correct in that the state string changes from using a thousands delimiter to no delimiter i.e. "1,500" -> "1500". Let me add that to the PR description too.

As to your other question I'll go ahead and assume that is targeted towards the other owners of the repo 😺

Reply:
If you do not apply formatting, why not just keep the integer?
```suggestion
                        self._attr_native_value = int(raw_state)"
```

Reply:
Good call - done in [06bb883](https://github.com/home-assistant/core/pull/85251/commits/06bb88347acf75b64846618ab7d72d7230c06a7b)